lib/whoop.rb

```diff
@@ -53,6 +56,8 @@ def whoop(
       explain: false,
       context: nil
     )
+      raise FormatNotSupportedError unless FORMATS.include?(format)
```

Comment:
I am glad you added this, although I wonder if we want to raise an error vs indicate to the developer that the format they provided was wrong.

The reason I say this is because this tool is to be used for debugging, and handling errors for a debugging tool may be annoying.

Instead, what if we added to the response (at the bottom) something like this:

```
NOTE: Invalid format option [#{format}]. Available formats: #{FORMATS.join(", ")})
```

Thoughts

Reply:
Totally fair, good shout! The error is unecessary friction